packages/-ember-caluma/package.json

```diff
@@ -14,6 +14,7 @@
   "devDependencies": {
     "@ember/jquery": "2.0.0",
     "@ember/optional-features": "2.0.0",
+    "@embroider/macros": "1.2.0",
```

Comment:
Why is the `^` not used in this `package.json` compared to the ones in the other packages?

Reply:
This is a dev dependency, not a dependency